packages/contracts-bedrock/contracts/dispute/FaultDisputeGame.sol

```diff
@@ -219,17 +300,23 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone {
      * @inheritdoc IDisputeGame
      */
     function resolve() external returns (GameStatus status_) {
+        // TODO: Do not allow resolution before clocks run out.
+
         if (status != GameStatus.IN_PROGRESS) {
+            // If the game is not in progress, it cannot be resolved.
             revert GameNotInProgress();
+        } else if (!claimData[0].countered) {
```

Comment:
I think this else case is already covered by the for loop. Is this just optimizing for the (hopefully) common case?

Reply:
This case isn't always covered; if the root claim was never contested, the loop will revert.

Reply:
What changed? [test_resolve_rootUncontested](https://github.com/ethereum-optimism/optimism/blob/37d0332ddc7547b2bdbf90d6bb16063cc2f8681c/packages/contracts-bedrock/contracts/test/FaultDisputeGame.t.sol#L287) covered this case before.

Reply:
![Screenshot 2023-06-21 at 6 54 08 PM](https://github.com/ethereum-optimism/optimism/assets/8406232/5c0756da-5449-4ac2-98bd-62d45ab0423b)

Reply:
ahh gotcha. Thanks for explaining.